scripts/onboard_definition_agent.py

```
@@ -0,0 +1,848 @@
+        trace_dir = REPO_ROOT / flashinfer_trace_dir
+    results = {}
+    found = list(trace_dir.glob(f"definitions/**/{definition}.json"))
+    op_type = found[0].parent.name if found else "unknown"
+    wf = trace_dir / "workloads" / op_type / f"{definition}.jsonl"
+    results["workload_exists"] = wf.exists()
+    results["workload_count"] = len(wf.read_text().splitlines()) if wf.exists() else 0
+    blobs = list((trace_dir / "blob" / "workloads" / op_type / definition).glob("*.safetensors"))
+    results["blob_count"] = len(blobs)
+    tf = trace_dir / "traces" / op_type / f"{definition}.jsonl"
+    if tf.exists():
+        records = [json.loads(l) for l in tf.read_text().splitlines() if l.strip()]
```

Ruff (0.15.9)

[error] Ambiguous variable name: `l`

(E741)